src/cryptoadvance/specter/cli/cli_noded.py

```diff
@@ -132,8 +132,8 @@ def bitcoind(
 )
 @click.option(
     "--port",
-    default=7040,
-    help="Specify a port the elementsd should run on. Default is 7040.",
+    default=18555,
```

Comment:
Is this really the default? 
Any changes to the default liquid RPC ports should also be reflected in the front end:
<img width="429" alt="grafik" src="https://user-images.githubusercontent.com/70536101/170037900-e87725f8-5bc1-44c0-b141-aa4b23598b1b.png">

Reply:
The default in the cli is not the default in specter. That's imho exactly the point because otherwise, you still have conflicts.
You should be able to long-run your (elements-) node how you would like. If you deal with the cli, it should not interefere, especially not with its default parameters.
Makes sense?

Reply:
No, it doesn't really make sense to me tbh. We've settled on default ports for elementsd and those should be reflected here as well. Non-default ports to avoid conflicts when running Cypress and Specter with bitcoind/elementsd are handled in test-cypress.sh.